app/spec/lib/bk/compat/__snapshots__/examples/github-actions/java-spring-boot-bmi.yaml.snap

```diff
@@ -35,14 +35,13 @@ steps:
 - commands:
   - "# action actions/checkout@v3 is not necessary in Buildkite"
   - echo '~~~ Add Server key'
-  - touch key.txt && echo "Context element secrets.SERVER_KEY can not be translated
-    (yet)" > key.txt
+  - touch key.txt && echo "$$SERVER_KEY" > key.txt
   - chmod 600 key.txt
   - echo '~~~ Deploy the application'
   - "("
-  - SERVER_HOST="Context element secrets.SERVER_HOST can not be translated (yet)"
-  - SERVER_PORT="Context element secrets.SERVER_PORT can not be translated (yet)"
-  - SERVER_USER="Context element secrets.SERVER_USER can not be translated (yet)"
+  - SERVER_HOST="$$SERVER_HOST"
```

Comment:
This looks weird and is redundant. I think we could fix that in a separate area to just not include the env vars and the subshell. CC @toote

Reply:
I agree it looks _dirty_, but it is an artifact of the small modular process of translating the original information as we never know where the variable is coming from (step commands are translated in one moment and contexts are translated in an isolated fashion - with no information as to where it is being translated on - at a later stage). It may be an interesting improvement, but a **very big** overhaul of the whole translation process.

Also note that secrets will also need manual intervention to setup plugins (in the step or the agent) to get those secrets available in the first place.

Reply:
Yeah okay makes sense